### DISCUSSION TITLE
how to enter once?

### DISCUSSION THREAD

**Discussion by Allen-c-HE** [acheong08/EdgeGPT#176] [Q&A]
Since v0.0.60, only 'escape+enter' can be used to send a message, requiring both hands. That's so inconvenient. And it seems that the option --enter-once doesn't work.
Is it possible to go back to enter once to send messages? Even enter twice is more convenient than the esc+enter.

**Comment by acheong08** [comment]
`alt+enter` also works

**Reply by acheong08** [reply]
I'll add --enter-once again

**Comment by acheong08** [comment]
https://github.com/acheong08/EdgeGPT/releases/tag/0.1.3

**Comment by Allen-c-HE** [comment]
Thanks!
I run the code with bat. Alt+enter means fullscreem.
